package.json

```diff
@@ -1,17 +1,31 @@
 {
   "name": "eslint-config-oclif",
   "description": "eslint config for oclif",
-  "version": "5.2.2",
+  "version": "5.2.3-dev.1",
   "author": "Salesforce",
   "bugs": "https://github.com/oclif/eslint-config-oclif/issues",
   "dependencies": {
+    "@eslint/compat": "^1.2.5",
+    "@eslint/eslintrc": "^3.2.0",
+    "@eslint/js": "^9.18.0",
+    "@typescript-eslint/eslint-plugin": "^8",
+    "@typescript-eslint/parser": "^8",
+    "eslint-config-oclif": "^5.2.2",
+    "eslint-config-xo": "^0.46.0",
     "eslint-config-xo-space": "^0.35.0",
+    "eslint-import-resolver-typescript": "^3.7.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-jsdoc": "^50.6.1",
     "eslint-plugin-mocha": "^10.5.0",
-    "eslint-plugin-n": "^15.1.0",
-    "eslint-plugin-unicorn": "^48.0.1"
+    "eslint-plugin-n": "^17.15.1",
+    "eslint-plugin-perfectionist": "^4",
+    "eslint-plugin-unicorn": "^56.0.1",
+    "typescript-eslint": "^8.20.0"
   },
   "devDependencies": {
-    "eslint": "^8.57.1"
+    "@oclif/prettier-config": "^0.2.1",
+    "eslint": "^9",
+    "typescript": "^5"
   },
   "engines": {
     "node": ">=18.0.0"
```

Comment:
increase minimum node to v18.18.0:
https://eslint.org/docs/latest/use/migrate-to-9.0.0#-nodejs--v1818-v19-are-no-longer-supported